docs/FileFormat.md

```diff
@@ -0,0 +1,132 @@
+# Input/Output files format
+
+## GMM
+### Input
+  D k n</br>
+  α<sub>1</sub></br>
+    ...</br>
+  α<sub>k</sub></br>
+  μ<sub>1,1</sub> ... μ<sub>D,1</sub></br>
+    ...</br>
+  μ<sub>1,k</sub> ... μ<sub>D,k</sub></br>
+  q<sub>1,1</sub> ... q<sub>D,1</sub> l<sub>1,1</sub> ... l<sub><sup>D(D-1)</sup>&frasl;<sub>2</sub>,1</sub></br>
```

Comment:
In my opinion, using `&frac12;D(D - 1)` would be beter, I mean l<sub>&frac12;D(D - 1), 1</sub> looks better than l<sub><sup>D(D-1)</sup>&frasl;<sub>2</sub>,1</sub>

Reply:
Fixed